packages/front-end/components/Experiment/TabbedPage/HealthTab.tsx

```diff
@@ -0,0 +1,207 @@
+import { DEFAULT_SRM_THRESHOLD } from "@/pages/settings";
+import { useSnapshot } from "../SnapshotProvider";
+import ExperimentDateGraph, {
+  ExperimentDateGraphDataPoint,
+} from "../ExperimentDateGraph";
+
+export interface Props {
+  experiment: ExperimentInterfaceStringDates;
+  onDrawerNotify: () => void;
+}
+
+const numberFormatter = new Intl.NumberFormat();
+
+const UnitCountDateGraph = ({
+  trafficByDate,
+  variations,
+}: {
+  trafficByDate: ExperimentSnapshotTrafficDimension[];
+  variations: ExperimentReportVariation[];
+}) => {
+  const [cumulative, setCumulative] = useState(false);
+  const { settings } = useUser();
+
+  const srmThreshold = settings.srmThreshold ?? DEFAULT_SRM_THRESHOLD;
+
+  // Get data for users graph
+  const usersPerDate = useMemo<ExperimentDateGraphDataPoint[]>(() => {
+    // Keep track of total users per variation for when cumulative is true
+    const total: number[] = [];
+    const sortedTraffic = [...trafficByDate];
+    sortedTraffic.sort((a, b) => {
+      return getValidDate(a.name).getTime() - getValidDate(b.name).getTime();
+    });
+
+    return sortedTraffic.map((d) => {
+      return {
+        d: getValidDate(d.name),
+        variations: variations.map((variation, i) => {
+          const users = d.variationUnits[i] || 0;
+          total[i] = total[i] || 0;
+          total[i] += users;
+          const v = cumulative ? total[i] : users;
+          const v_formatted = v + "";
+          return {
+            v,
+            v_formatted,
+            label: numberFormatter.format(v),
+          };
+        }),
+        srm: d.srm,
+      };
+    });
+  }, [trafficByDate, variations, cumulative]);
+
+  return (
+    <>
+      <div className="mt-3 mb-3 d-flex align-items-center">
+        <div className="mr-3">
+          <strong>Graph Controls: </strong>
+        </div>
+        <div>
+          <Toggle
+            label="Cumulative"
+            id="cumulative"
+            value={cumulative}
+            setValue={setCumulative}
+          />
+          Cumulative
+        </div>
+      </div>
+
+      <div className="mt-2 mb-2">
+        <ExperimentDateGraph
+          yaxis="users"
+          variationNames={variations.map((v) => v.name)}
+          label="Users"
+          datapoints={usersPerDate}
+          tickFormat={(v) => numberFormatter.format(v)}
+          srmThreshold={cumulative ? undefined : srmThreshold}
+        />
+      </div>
+    </>
+  );
+};
+
+export default function HealthTab({ experiment, onDrawerNotify }: Props) {
+  const { error, snapshot, phase } = useSnapshot();
+  const { runHealthTrafficQuery } = useOrgSettings();
+  const permissions = usePermissions();
+  const hasPermissionToEditOrgSettings = permissions.check(
+    "organizationSettings"
+  );
+
+  // If org has not updated settings since the health tab was introduced, prompt the user
+  // to enable the traffic query setting
+  if (runHealthTrafficQuery === undefined) {
```

Comment:
For now we probably also want to catch the case where it is set to false and have very similar logic.

"Health queries are disabled in your Organization Settings. To enable them, {SAME LOGIC YOU USE FOR UNDEFINED}"